components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -241,30 +261,66 @@ function TeamUsage() {
+                                                >
+                                                    <div className="flex flex-col col-span-2 my-auto">
+                                                        <span className="text-gray-700 dark:text-gray-400">
+                                                            {getType(usage.workspaceType)}
+                                                        </span>
+                                                        <span className="text-sm text-gray-400 dark:text-gray-600">
+                                                            {usage.workspaceClass}
+                                                        </span>
+                                                    </div>
+                                                    <div className="flex flex-col col-span-5 my-auto">
+                                                        <span className="truncate text-gray-700 dark:text-gray-400">
+                                                            {usage.workspaceId}
+                                                        </span>
+                                                        <span className="text-sm truncate text-gray-400 dark:text-gray-600">
+                                                            {toRemoteURL(usage.contextURL)}
+                                                        </span>
+                                                    </div>
+                                                    <div className="flex flex-col my-auto">
+                                                        <span className="text-right text-gray-700 dark:text-gray-400">
+                                                            {usage.credits.toFixed(1)}
+                                                        </span>
+                                                        <span className="text-right truncate text-sm text-gray-400 dark:text-gray-600">
+                                                            {getMinutes(usage)}
+                                                        </span>
+                                                    </div>
+                                                    <div className="my-auto" />
+                                                    <div className="flex flex-col col-span-3 my-auto">
+                                                        <span className="text-gray-400 truncate">
+                                                            {displayTime(usage.startTime)}
+                                                        </span>
+                                                        <div className="flex">
+                                                            {usage.workspaceType === "prebuild" ? (
+                                                                <UsageIcon className="my-auto" />
+                                                            ) : (
```

Comment:
Btw this icon doesn't have that outer layer, @gtsiolis. I couldn't click on the outer selection in Figma as we saw that other time. Feel free if you'd like to send me the correct SVG 🙏🏽